core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -2360,6 +2361,27 @@ private void checkNodeTypeCount(String sql, Map<Class<? extends RelNode>, Intege
     checkNodeTypeCount(sql, expected);
   }
 
+  @Test public void testNodeTypeExchange() {
```

Comment:
Other tests for node type checking is by `testNodeTypeXXX `, shall we follow the convention ?